### ISSUE TITLE
Error AS couldn't recover the platforms

### ISSUE THREAD

**Issue by kinow** [BSC-ES/autosubmit#677]
In GitLab by @acarreri on Mar 30, 2021, 12:09

Hi @mcastril, @dbeltrankyl 

A weird error just happened on AS01 machine. For two different experiments, on my console I have:
~~~
[ERROR] Trace:
[ERROR] Command sacct -n -X --jobs  15073477 -o jobid,State in mn1.bsc.es warning:  [eCode=6005]
Waiting 30 seconds before continue
Creating jobs...
Loading JobList
Adding dependencies...
Removing redundant dependencies...
Using existing valid structure.
Checking the connection to all platforms in use
[local] Connection successfull to host localhost
[local] Correct user privileges for host localhost
[marenostrum4] Connection successfull to host mn1.bsc.es
[nord3] Connection successfull to host nord1.bsc.es
[nord3] Correct user privileges for host nord1.bsc.es
[transfer_node] Connection successfull to host dt01.bsc.es
[transfer_node] Correct user privileges for host dt01.bsc.es
[ERROR] Trace: Command sacct -n -X --jobs  15073477 -o jobid,State in mn1.bsc.es warning:
 [CRITICAL] Autosubmit couldn't recover the platforms [eCode=7050]
~~~

On the GUI, the experiments appear as `INACTIVE` but with one RUNNING job in green and this job is running on MN4.

I don't know what it means. I will relaunch AS but I prefer to let you know of this behavior.

Thanks

**Comment by kinow** [comment]
In GitLab by @dbeltrankyl on Mar 30, 2021, 12:20

We had filesystem errors a while ago, 

In 3.13.0b, we have a mechanism to do a bunch of retrials in case of platform failure. (today we had filesystem issues)

However, there is a hard limit to avoid endless retrials and I think that is what could happened, another reason could be an incorrect configuration or if there was an i/o issue together with a platform issue ( since it couldn't retrieve the user info) 

Based on the info of the checkers:
```
[local] Connection successfull to host localhost
[local] Correct user privileges for host localhost
[marenostrum4] Connection successfull to host mn1.bsc.es
[nord3] Connection successfull to host nord1.bsc.es
[nord3] Correct user privileges for host nord1.bsc.es
[transfer_node] Connection successfull to host dt01.bsc.es
[transfer_node] Correct user privileges for host dt01.bsc.es
```

there is a missing line on marenostrum4, so there was a failure on that check. The message should be different, so I will check why is not displaying the correct error message.

**Comment by kinow** [comment]
In GitLab by @dbeltrankyl on Mar 30, 2021, 12:26

Oh, I see that the last checker added, also added a bug on that mechanism, allowing only one(1min) reconnection instead of 120(2hours). 

Fixing it, thanks @acarreri .

FYI: @mcastril

**Comment by kinow** [comment]
In GitLab by @acarreri on Mar 30, 2021, 12:28

Thanks @dbeltrankyl 

I can relaunch the experiments normally now, right?

**Comment by kinow** [comment]
In GitLab by @dbeltrankyl on Mar 30, 2021, 12:28

Yes

**Comment by kinow** [comment]
In GitLab by @dbeltrankyl on Mar 30, 2021, 12:53

fixed via 0f53a367a9f22d5805f6de516eee32beb98549e2 will be uploaded later today
